### ISSUE TITLE
 not working on 3.2.3

### ISSUE THREAD

**Issue by amien90** [fadyheiba/ShowHide#4]
can anyone confirm what this extension is not working on 3.2.3?

**Comment by ManuelRodri** [comment]
Yes, the extension is awesome, I usually use it in my dashboards, but it doesn't work in 3.2.3

**Comment by ManuelRodri** [comment]
I'm not so good with JS and the API for QS, I've only detected that problem is in line 80 of FEI-ShowHide.js

		$scope.currentChartModel.enigmaModel.endSelections(true)

Error is:

                TypeError: Cannot read property 'endSelections' of undefined

**Comment by ManuelRodri** [comment]
Hi @amien90 meanwhile @fadyheiba give a solution I changed the function renderChart() skipping the part of delete the old object (it contains the error line)

 function renderChart() {
                    $scope.app.getObject($element.find('div'), $scope.currentChart).then(function(model) {
                        $scope.currentChartModel = model;
                    });
            };


Is not an ideal solution but is compatible with this QS version.
